systempony

```diff
@@ -90,13 +102,20 @@ function ponyget_CPU_Usage()
 
 function ponyget_Uptime()
 {
-    uptime | grep -oE "up.*user" | \
-        sed -r -e "s/up\s*(.+),\s*[0-9]+\s*user/\1/" -e "s/\s+/ /g"
+    let uptime=$(awk -F. '{print $1}' /proc/uptime)
+    let sec="$uptime%60"
+    let uptime/=60
+    let min="$uptime%60"
+    let uptime/=60
+    let hrs="$uptime%24"
+    let days="$uptime/24"
+
+    printf "%d days, %d:%02d:%02d" $days $hrs $min $sec
```

Comment:
Not sure I like the output format of this one. AFAIK `busybox uptime` can be given to the pipeline I had here and it will give the same output as with GNU `uptime`

Reply:
The Busybox output is formatted as `06:40:53 up  6:24,  load average: 0.01, 0.03, 0.00`